src/test/java/com/pubnub/api/endpoints/pubsub/SignalTest.java

```diff
@@ -126,11 +128,13 @@ public void onResponse(PNPublishResult result, @NotNull PNStatus status) {
 
     @Test
     public void testSignalSuccessReceive() {
+        final String expectedUserMessageTypeValue = "myMessageType";
+        final String expectedSpaceIdValue = "1-to-1_chat";
 
         stubFor(get(urlMatching("/v2/subscribe/mySubscribeKey/coolChannel/0.*"))
                 .willReturn(aResponse().withBody("{\"m\":[{\"c\":\"coolChannel\",\"f\":\"0\",\"i\":\"uuid\"," +
                         "\"d\":\"hello\",\"e\":1,\"p\":{\"t\":1000,\"r\":1},\"k\":\"mySubscribeKey\"," +
-                        "\"b\":\"coolChannel\"}],\"t\":{\"r\":\"56\",\"t\":1000}}")));
+                        "\"b\":\"coolChannel\",\"mt\":\"" + expectedUserMessageTypeValue + "\",\"si\":\"" + expectedSpaceIdValue + "\"}],\"t\":{\"r\":\"56\",\"t\":1000}}")));
```

Comment:
Good catch.  The reason why test here doesn't have helper method is that I modified it whereas code that uses getSubscribeResponseForSignalWithMessageTypeAndSpaceId() is written from scratch. 
I will modify testSignalSuccessReceive() to use helper method. I think this might add more clarity to this test.